apps/crc_interactive.py

```diff
@@ -81,9 +81,15 @@ def _validate_arguments(self, args: Namespace) -> None:
             args: Parsed commandl ine arguments
         """
 
-        # Check wall time is between limits
-        if not self.min_time <= args.time <= self.max_time:
-            self.error(f'{args.time} is not in {self.min_time} <= time <= {self.max_time}... exiting')
+        # Check wall time is between limits, enable both %H:%M format as well as integer hours
+        check_time=0
+        if args.time.find(':')<0:
+            check_time=int(args.time)
+        else:
+            check_time=datetime.strptime(args.time,'%H:%M').hour
```

Comment:
This will ignore the minutes values. For example:

```python
datetime.strptime('2:30', '%H:%M').hour
```

Returns `2`, not `2.5`.

Reply:
Yes, I figured that since we are treating hours as integers here we could round up or down, thus ignoring the minutes.